docs/src/pages/style/typography/typography.md

```diff
@@ -59,10 +69,12 @@ It’s important to realize that the style of a typography is independent from t
 - You can change the underlying element for a one time occassion with the `component` property:
 
 ```jsx
-{/* We already have an h1 in the page, let's not duplicate it. */}
+{
+  /* We already have an h1 in the page, let's not duplicate it. */
```

Comment:
?

Reply:
I'm really sorry, more unknown origin changes. This one looks like it came from VSCode, but I haven't seen it do these kind of changes before. Will Revert